awsx/ec2/vpc.ts

```diff
@@ -218,6 +223,7 @@ export class Vpc extends schema.Vpc<VpcData> {
                 allocationId: createEip ? eips[i].allocationId : allocationIds[i],
                 tags: {
                   Name: `${name}-${i + 1}`,
+                  ...args.tags,
```

Comment:
This should be before the name to ensure Name doesn't get overwritten.

Reply:
Fixed!